src/test/java/org/jenkinsci/plugins/workflow/support/steps/ExecutorStepDynamicContextTest.java

```diff
@@ -207,4 +215,36 @@ public class ExecutorStepDynamicContextTest {
         });
     }
 
+    @Test public void onceRetentionStrategyNodeDisappearance() throws Throwable {
```

Comment:
Essentially a copy of `normalNodeDisappearance` checking some behavioural differences.